thirdparty_builtin/CMakeLists.txt

```diff
@@ -57,18 +57,36 @@ if(ENABLE_TESTS)
         list(APPEND gtest_defines "-DGTEST_HAS_DEATH_TEST=0")
     endif()
 
+    #
+    # Use BLT's ENABLE_GMOCK option to set the build options for 
+    # gtest's cmake project
+    #
+    
+    if(ENABLE_GMOCK)
```

Comment:
Check out how they are used in:

https://github.com/LLNL/blt/blob/master/thirdparty_builtin/googletest-release-1.8.0/CMakeLists.txt

While keeping in mind that building gmock also takes care of building gtest.